WeakAuras/SubRegionTypes/SubBorder.lua

```diff
@@ -0,0 +1,88 @@
+if not WeakAuras.IsCorrectVersion() then return end
```

Comment:
Oh, no no no no. We aren’t doing this, please. I relented when @InfusOnWoW named the text subregion filed as SubText, but I am *not* pleased with the idea of naming every damn file in this directory SubSomething. That’s really dumb.

It’s a border, so just call it Border.lua

Reply:
I agree :D